### PR TITLE
doc: Added a few details to generic upgrade guide

### PR DESCRIPTION
**Pull request by rkothiya**
Fixes: #1364

Signed-off-by: Rinku Kothiya <rkothiya@redhat.com>

### CODE REVIEW

File: docs/Upgrade-Guide/Generic_Upgrade_procedure.md [hunk 1]
```diff
@@ -24,7 +24,23 @@ This procedure involves upgrading **one server at a time**, while keeping the vo
...
+
+        # cat /etc/yum.d/newglusterrepo.repo 
+         [newglusterrepo]
+         name=NewGlusterRepo
+         baseurl="file:///new-gluster-rpms/"
+         gpgcheck=0
+
+    3.3  Upgrade glusterfs, for example to upgrade glusterfs-server to x.y version : 
+    
+        # yum install glusterfs-server-x.y.fc30.x86_64.rpm
```
Comment (kshithijiyer) [inline docs/Upgrade-Guide/Generic_Upgrade_procedure.md:43]: Shouldn't this be `yum update` instead?

File: docs/Upgrade-Guide/Generic_Upgrade_procedure.md [hunk 2]
```diff
@@ -24,7 +24,23 @@ This procedure involves upgrading **one server at a time**, while keeping the vo
...
+
+        # createrepo /new-gluster-rpms/
+
+    3.2  Create the .repo file in /etc/yum.d/ : 
+
+        # cat /etc/yum.d/newglusterrepo.repo 
+         [newglusterrepo]
+         name=NewGlusterRepo
+         baseurl="file:///new-gluster-rpms/"
+         gpgcheck=0
```
Comment (kshithijiyer) [inline docs/Upgrade-Guide/Generic_Upgrade_procedure.md:39]: Add one more param in the file `enabled=1` so the complete thing should be as below:
```
        # cat /etc/yum.d/newglusterrepo.repo 
         [newglusterrepo]
         name=NewGlusterRepo
         baseurl="file:///new-gluster-rpms/"
         gpgcheck=0
         enabled=1
```

File: docs/Upgrade-Guide/Generic_Upgrade_procedure.md (hunk 1 above)
Comment (rkothiya) [reply docs/Upgrade-Guide/Generic_Upgrade_procedure.md:43]: Done

File: docs/Upgrade-Guide/Generic_Upgrade_procedure.md (hunk 2 above)
Comment (rkothiya) [reply docs/Upgrade-Guide/Generic_Upgrade_procedure.md:39]: Done

**Pull request merged**